generator/main.go

```diff
@@ -50,6 +58,7 @@ var (
 		Name: "num_events_ingested",
 		Help: "Number of events ingested to the Destination",
 	})
+	r = rand.New(rand.NewSource(0))
```

Comment:
from the go docs it seems [NewSource](https://pkg.go.dev/math/rand#NewSource), the source is not safe for concurrent use - will that be a problem here?

Reply:
I made some new random sources local to a particular go routine just in case.